config/spryk/spryks/Spryker/Zed/Communication/Plugin/AddCheckoutDoSaveOrderPlugin.yml

```diff
@@ -0,0 +1,32 @@
+spryk: template
+description: "Adds a new *module* CheckoutPostSave class."
+mode: both
+
+arguments:
+    module:
+        inherit: true
+
+    organization:
+        inherit: true
+        default: SprykerEco
+
+    className:
+        value: "{{ paymentMethod }}CheckoutDoSaveOrderPlugin"
+
+    implementerInterfaceName:
+        value: \Spryker\Zed\CheckoutExtension\Dependency\Plugin\CheckoutDoSaveOrderInterface
+
+    expanderClassName:
+        value: "Spryker/Zed/Kernel/Communication/AbstractPlugin"
+
+    subDirectory:
+        value: "Zed/{{ module }}/Communication/Plugin/Checkout"
```

Comment:
can you use the same structure for spryks 
Plugin/Oms
Plugin/Checkout
Plugin/Payment

Reply:
changed